app.rb

```diff
@@ -1,2 +1,55 @@
 require 'sinatra'
-require 'sinatra/reloader'
\ No newline at end of file
+require 'sinatra/reloader'
+
+# # 99 Bottles of Beer
+#
+# ## Project Specs
+#
+# - The home page `/` should list:
```

Comment:
This suggests that you need to keep something for the `'/'` route. To get the bonus to work you tell sinatra to redirect to `'/99'` when someone visits just `'/'`
[Check out the documentation](http://www.sinatrarb.com/intro.html#Browser%20Redirect)